packages/types/src/proof/semaphoreProof.ts

```diff
@@ -1,25 +1,16 @@
 import { FullProof } from "@semaphore-protocol/proof";
-import { MerkleProof } from "@zk-kit/incremental-merkle-tree";
 
-import { MerkleProofArtifacts } from "./merkle";
+import { ZkCircuit, ZkInputs } from "./zkProof";
 
-export interface SemaphoreProof {
-  fullProof: FullProof;
-}
+export type SemaphoreFullProof = FullProof;
 
-export interface SemaphoreProofRequest {
-  identitySerialized: string;
+export interface ISemaphoreProofRequest extends ZkCircuit, ZkInputs {
+  identitySerialized?: string;
```

Comment:
Why does it become optional?

Reply:
If user doesn't operate with type directly and this one is only the internal type, I'm good with it.

Reply:
It's fixed.